sdk/keyvault/keyvault-common/src/parseKeyvaultIdentifier.ts

```diff
@@ -17,6 +20,10 @@ export interface ParsedKeyVaultEntityIdentifier {
    */
   name: string;
 }
+
+/**
+ * Parses a Key Vault identifier into its components.
```

Comment:
Add ref doc comments for parameters?

Reply:
Good call. I'm also going to rename this function to `parseKeyVaultIdentifier`, the casing is inconsistent with the other functions.